src/components/LoanCards/BorrowerInfo/BorrowerInfoBody.vue

```diff
@@ -50,6 +50,10 @@ export default {
 			type: String,
 			default: ''
 		},
+		maxUseLength: {
+			type: Number,
+			default: 500
```

Comment:
Why did we reduce the default to 500 for GridLoanCard and ListLoanCard?

Reply:
Just wanted to put a more "sane" default in case there are real long loan use statements out there it would keep it from pushing that area too far down on the list item card.

Reply:
Checked my local db and there were 0 with 300+ characters or more, 97 with 200 chars or more.